src/page-weekly-snapshot/SystemWideProjectionChart.tsx

```diff
@@ -119,7 +142,7 @@ const SystemWideProjectionChart: React.FC = () => {
         orient: "bottom",
         baseline: "under",
         label: "Date",
-        tickValues: chartData.map((d) => d.date),
+        tickValues: chartData[0].coordinates.map((d) => d.date),
```

Comment:
This is guaranteed to be non-empty at this point, right?

Reply:
It shouldn't be since we are showing the missing scenario data text if there isn't data, but I'll add a guard just in case